html/semantics/embedded-content/the-img-element/relevant-mutations.html

```diff
@@ -159,43 +159,43 @@
 
   t('crossorigin absent to empty', function(img) {
     img.crossOrigin = '';
-  }, 'error');
+  }, 'timeout');
 
   t('crossorigin absent to anonymous', function(img) {
     img.crossOrigin = 'anonymous';
-  }, 'error');
+  }, 'timeout');
 
   t('crossorigin absent to use-credentials', function(img) {
     img.crossOrigin = 'use-credentials';
-  }, 'error');
+  }, 'timeout');
 
   t('crossorigin empty to absent', function(img) {
     img.removeAttribute('crossorigin');
-  }, 'error');
+  }, 'timeout');
 
   t('crossorigin empty to use-credentials', function(img) {
     img.crossOrigin = 'use-credentials';
-  }, 'error');
+  }, 'timeout');
```

Comment:
There is an `img` in the markup for each test, with a `data-desc` attribute that matches the first argument to `t`. The tests are all started on page load and finish individually when the test's `img` gets a `load` or `error` event, or times out.

For this test, the markup is
```
<img crossorigin data-desc="crossorigin empty to use-credentials">
```

Note no `src` or `srcset` attribute.